### ISSUE TITLE
Create the Setting Screen that allows editing the display language of the user and the application

### ISSUE THREAD

**Issue by cristianMarcial** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#185]
## Description
A setting screen will be created that allows the user to edit the display language of their own account and the local one (i.e. the displayed language before the user logs in). This setting screen can only be accessed when they have already logged in, since it is not possible to edit the language of a user who has not even logged in.

## Objective
Give the logged in user the ability to change the language of the application.

## Requirements
- This screen must be accessible and visible to the user. That is, there must be a button on one of the screens that allows access to this screen.
- It must have buttons that allow changing the language.

## Time Constrains & Deadline
Preferably, not before November 22nd.

## Difficulty
8, since it requires knowing how to manage the application directories and knowledge regarding the graphics libraries being used.

## Priority
5

## Suggested Asignees:
@Yamil-Serrano, @cristianramos9

**Comment by Yamil-Serrano** [comment]
I have currently managed to add a test button with dynamic icons that change depending on the language you select in the user profile section. It should be noted that the button does not yet have any effect on the user interface to change the text to the desired language.

https://github.com/user-attachments/assets/901f9174-26dd-4128-8baf-ef52654255cc

**Comment by Yamil-Serrano** [comment]
@cristianMarcial, @cristianramos9 I've created an updated version where the language change button is now positioned at the bottom right corner of the window. The button was implemented in `RegiUPR.py`, and a method has been added to make the button visible or invisible. This method can be used in any window to facilitate easy access to the button's state, making it simpler to integrate with the `update_ui_language` function.

![image](https://github.com/user-attachments/assets/928858a5-b1c2-42c1-bd39-eb1a48e067f1)

![image](https://github.com/user-attachments/assets/52033aa9-bb81-4a29-be67-d42b0c40d39c)
